WireAPI/Sources/WireAPI/Models/API/APIVersion.swift

```diff
@@ -44,7 +44,7 @@ public enum APIVersion: UInt, CaseIterable, Comparable, Sendable {
     /// Only if these critera are met should we explicitly mark the version
     /// as production ready.
 
-    public static let productionVersions: Set<Self> = [.v0, .v1, .v2, .v3, .v4, .v5, .v6]
+    public static let productionVersions: Set<Self> = [.v0, .v1, .v2, .v3, .v4, .v5, .v6, .v7]
```

Comment:
I noticed that this was out of sync with the `WireTransport.APIVersion`, but it didn't cause any issues since resolving was until now done via the `WireTransport` version.